src/components/common/UserProfileEdit/UserAdvancedSettings.tsx

```diff
@@ -83,23 +73,39 @@ const headingTextValues = {
   ),
 };
 
-const UserAdvancedSettings = () => {
+interface Props {
+  user: User;
+}
```

Comment:
Alternatively the current user could be retrieved from the `useAppContext` hook, especially since you already call it

Reply:
This follows the pattern in `UserMainSettings` of passing down the User object as a prop.

I believe this is so that the possibility of the object being `null` or `undefined` is handled by the parent component:

(https://github.com/JoinColony/colonyCDapp/blob/3d206b51e22e22e016c1ea7a08203951ebf1b686/src/components/common/UserProfileEdit/UserProfileEdit.tsx#L36C1-L41)

otherwise it gets really annoying to constantly be dealing with the null / undefined possibility